iris/download.html

```diff
@@ -33,25 +33,34 @@
         
             <li><a href="index.html">Home</a></li>
             <li><a href="download.html">Download</a></li>
-            <li><a href="documentation.html">Documentation</a></li>
+            <li><a href="docs/latest/index.html">Documentation</a></li>
+            <li><a href="documentation.html">Versions</a></li>
             <li><a href="community.html">Community</a></li>
             <li><a href="../governance.html">Governance</a></li>
             <li><a href="code.html">Code</a></li>
-            <li><a href="../index.html">Scitools</a></li>
+            <li><a href="../index.html">SciTools</a></li>
 
         </ul>
     </div>
 
     
 
-     <div id="bd">
-            <h1>Download</h1>
-            <h2>Official releases</h2>
-             <p>Official releases can be found <a href="https://github.com/SciTools/iris/tags">here</a>. These releases contain source, documentation and instructions on how to build and install Iris under Linux.</p>
-            <h2>GitHub repository</h2>
-             <p>We strongly encourage people to contribute to Iris and for this type of development we suggest you obtain the latest version of the code from our github repository. Assuming you have <a href="http:git-scm.org">git</a> installed, enter the following command into a terminal:</p>
-        </p><pre class="code">git clone https://github.com/SciTools/iris.git</pre>
```

Comment:
There was a spurious closing `p` tag on this line. In fixing it I added some indentation to the html and liked it, so I've left it in place.